### PR TITLE
Mchoi/wpartialasides

### PR DESCRIPTION
**Pull request by Meahaa**
### Summary

Remove aside titles from remaining partials with folders with w titles

### CODE REVIEW

File: src/content/partials/waf/managed-lists.mdx [hunk 1]
```diff
@@ -12,6 +12,6 @@
 
 <br />
 
-:::caution[<a id="vpn-important-remarks"></a>Important remarks about the Cloudflare VPNs Managed List]
```
Comment (pedrosousa) [inline src/content/partials/waf/managed-lists.mdx:15]: We need to keep the anchor because this aside is the target of a link in a different page.

Comment (Meahaa) [reply src/content/partials/waf/managed-lists.mdx:15]: makes sense! I'll add it to the body content then since I'm removing the headers

**Review by Meahaa — COMMENTED** [review]
update managed-lists

**Comment by Meahaa** [comment]
duplicate

**Pull request closed without merging**